Scenarios/CGCT/CGCT/CGView.m

```diff
@@ -18,16 +18,16 @@
 #import "CGView.h"
 
 @implementation CGView {
-    DemoTableViewCell* demoToDraw;
+    DemoScenario* demoToDraw;
 };
 
-- (void)updateCurrentDemo:(DemoTableViewCell*)newDemo {
+- (void)updateCurrentDemo:(DemoScenario*)newDemo {
```

Comment:
if you just make this a property setter, backing `@property DemoScenario* demo;`, you'll get updates for free...